cluster/aws/util.sh

```diff
@@ -1383,39 +1549,41 @@ function wait-master() {
 function check-cluster() {
   echo "Sanity checking cluster..."
 
-  sleep 5
-
-  detect-nodes > $LOG
+  if [[ "${PRESET_INSTANCES_ENABLED:-}" != "yes" ]]; then
+    sleep 5
+    
+    detect-nodes > $LOG
 
-  # Don't bail on errors, we want to be able to print some info.
-  set +e
+    # Don't bail on errors, we want to be able to print some info.
+    set +e
 
-  # Basic sanity checking
-  # TODO(justinsb): This is really not needed any more
-  local rc # Capture return code without exiting because of errexit bash option
-  for (( i=0; i<${#KUBE_NODE_IP_ADDRESSES[@]}; i++)); do
-      # Make sure docker is installed and working.
-      local attempt=0
-      while true; do
-        local minion_ip=${KUBE_NODE_IP_ADDRESSES[$i]}
-        echo -n "Attempt $(($attempt+1)) to check Docker on node @ ${minion_ip} ..."
-        local output=`check-minion ${minion_ip}`
-        echo $output
-        if [[ "${output}" != "working" ]]; then
-          if (( attempt > 20 )); then
-            echo
-            echo -e "${color_red}Your cluster is unlikely to work correctly." >&2
-            echo "Please run ./cluster/kube-down.sh and re-create the" >&2
-            echo -e "cluster. (sorry!)${color_norm}" >&2
-            exit 1
+    # Basic sanity checking
```

Comment:
I don't see any changes here, why are those marked as updated. Hard to review.

Reply:
They are included in "if" so there is empty space added for each line.

Reply:
Most of the time, UI shows empty spaces diff correctly. In this case, it shows the entire line as replaced. That's what makes the code review harder.

Reply:
Updated

Reply:
This is not ideal